RushilDaya-BBD/IdiomOfTheDay
# Feat/endpoint config

Completed login, logout and token endpoints securely. 

To test:
- navigate to http://localhost:8080/admin
- you should be automatically navigated to the login page. 
- After logging in, you'll be redirected back to http://localhost:8080/admin. See the token in the console.
- There is a dummy variable "logoutButtonClicked" in the code for now. If logoutButtonClicked === true, you will be logged out soon after logging in. Log out will direct you to http://localhost:8080/ . If logoutButtonClicked === false, you won't log out i.e. you won't be redirected.
